### ISSUE TITLE
delete a contact's message (not mine) for everyone (feature request)

### ISSUE THREAD

**Issue by Wjxfi** [simplex-chat/simplex-chat#1767]
This is a very useful feature. Telegram uses it well, and no one is complaining.

>concept:
>
this could be included in the ```contact preferences``` (to be allowed, you need to enable "delete for all" before doing so).

_I need it._

**Comment by epoberezkin** [comment]
It’s interesting for groups in particular - to allow admins to delete other people messages. We will think more about it.

**Comment by valtoree** [comment]
Yes I definitely need this if I'm to leave riot/matrix as they have it.
